Grafana/confd/templates/lambdas-dashboard.json.tpl

```diff
@@ -0,0 +1,611 @@
+                "5m",
+                "now"
+              ]
+            },
+            "reducer": {
+              "params": [],
+              "type": "avg"
+            },
+            "type": "query"
+          }
+        ],
+        "executionErrorState": "alerting",
+        "for": "10m",
+        "frequency": "1m",
+        "handler": 1,
+        "message": "Lambda function ${Label} failed",
+        "name": "Other Lambdas Error Alert",
+        "noDataState": "ok",
+        "notifications": []
+      },
+      "datasource": null,
+      "fieldConfig": {
+        "defaults": {
+          "color": {
+            "mode": "palette-classic"
+          },
+          "custom": {
+            "axisLabel": "",
+            "axisPlacement": "auto",
+            "barAlignment": 0,
+            "drawStyle": "line",
+            "fillOpacity": 0,
+            "gradientMode": "none",
+            "hideFrom": {
+              "legend": false,
+              "tooltip": false,
+              "viz": false
+            },
+            "lineInterpolation": "linear",
+            "lineWidth": 1,
+            "pointSize": 5,
+            "scaleDistribution": {
+              "type": "linear"
+            },
+            "showPoints": "auto",
+            "spanNulls": false,
+            "stacking": {
+              "group": "A",
+              "mode": "none"
+            },
+            "thresholdsStyle": {
+              "mode": "off"
+            }
+          },
+          "mappings": [],
+          "thresholds": {
+            "mode": "absolute",
+            "steps": [
+              {
+                "color": "green",
+                "value": null
+              },
+              {
+                "color": "red",
+                "value": 80
+              }
+            ]
+          }
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 12,
+        "x": 0,
+        "y": 16
+      },
+      "id": 10,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom"
+        },
+        "tooltip": {
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "targets": [
+        {
+          "datasource": null,
+          "editorMode": "code",
+          "expr": "aws_lambda_errors_sum{function_name=\"bichard-7-{{getv "/cjse/infra/envname" ""}}-archive-user-logs\"}",
+          "format": "time_series",
+          "range": true,
+          "refId": "Archive user logs"
+        },
+        {
+          "datasource": null,
+          "expr": "aws_lambda_errors_sum{function_name=\"bichard-7-{{getv "/cjse/infra/envname" ""}}-add-archival-events\"}",
+          "hide": false,
+          "refId": "Add archival events"
+        },
+        {
+          "datasource": null,
+          "expr": "aws_lambda_errors_sum{function_name=\"bichard-7-{{getv "/cjse/infra/envname" ""}}-retry-failed-messages\"}",
```

Comment:
we're counting for the number of times the lambda has failed here but the lambda currently just logs when some records failed and exits with normal status https://github.com/ministryofjustice/bichard7-next-audit-logging/blob/main/src/add-archival-events/src/addArchivalEvents.ts#L51

do we want to either change this to count the number of lines logged to STDERR or have the `add-archival-events` lambda exit with non-zero status when any errors occurred?